doc/modules/classes.rst

```diff
@@ -1512,23 +1512,4 @@ To be removed in 0.22
    :template: deprecated_function.rst
 
    covariance.graph_lasso
-   datasets.fetch_mldata
```

Comment:
`datasets.fetch_mldata` is still to be removed in 0.22, so it was fine I think.

Reply:
right... whoops.

Reply:
Wait, I didn't remove this. This is fine, right?

Reply:
You didn't indeed remove the code, but in `doc/modules/classes.rst`, `fetch_mldata` was removed from the list of classed "To be removed in 0.22". I think it sill belongs there?

Reply:
No sorry, it's bellow, missed it.

Reply:
Github did something strange, don't ask me ;)